Source/Dafny/DafnyPipeline.csproj

```diff
@@ -18,9 +18,11 @@
     <DefineConstants>TRACE</DefineConstants>
     <PackageVersion>1.1.0</PackageVersion>
     <TargetFramework>net5.0</TargetFramework>
+    <Nullable>enable</Nullable>
   </PropertyGroup>
 
   <ItemGroup>
+      <PackageReference Include="JetBrains.Annotations" Version="2021.2.0-eap3" />
```

Comment:
This is a NuGet package ([ref](https://www.nuget.org/packages/JetBrains.Annotations/)). It adds noop annotations that improve editor tooling when using Rider or ReSharper, but otherwise have no effect.